packages/nova-users/lib/helpers.js

```diff
@@ -232,8 +232,8 @@ Users.getProperty = function (object, property) {
   var array = property.split('.');
   if(array.length > 1){
     var parent = array.shift();
-    // if our property is not at this level, call function again one level deeper if we can go deeper, else return null
-    return (typeof object[parent] === "undefined") ? null : this.getProperty(object[parent], array.join('.'));
+    // if our property is not at this level, call function again one level deeper if we can go deeper, else return undefined
+    return (typeof object[parent] === "undefined") ? undefined : this.getProperty(object[parent], array.join('.'));
```

Comment:
If my setting was at false, the old way was returning `null`: the chaining of `getProperty` -> `getSetting` with the `null` + `_.isEmpty` doesn't return the good value.

That's why I switched to `undefined`